### ISSUE TITLE
invalid markdown

### ISSUE THREAD

**Issue by 3limin4t0r** [rubocop-hq/ruby-style-guide#727]
So the guide specifically says the following:

> You can generate a PDF or an HTML copy of this guide using [Pandoc][].

  [pandoc]: http://pandoc.org/

However when converting to PDF it crashes with the following error:

```bash
pandoc -s -o ruby-style-guide.pdf --latex-engine=xelatex ruby-style-guide.md
```

> ```
> ! You can't use `macro parameter character #' in math mode.
> l.3409 \(global = 0  #
> 
> pandoc: Error producing PDF
> ```

When converting to HTML the output content isn't correct either, but doesn't produce an error.

The above is most likely caused by the invalid indention of the lists and list content. The [markdown syntax documentation][] says the following:

> List items may consist of multiple paragraphs. Each subsequent paragraph in a list item must be indented by either 4 spaces or one tab:

The above specification is not followed in the [README.md][] and is often indented with 2 spaces.

  [markdown syntax documentation]: https://daringfireball.net/projects/markdown/syntax#list
  [readme.md]: ../blob/master/README.md

**Comment by bbatsov** [comment]
Does everything work if you change the indentation?

**Comment by 3limin4t0r** [comment]
It was quite a pain to fix the indention of the whole file (PR #729), but yes changing the indention from 2 to 4 spaces resolves this issue and generates a PDF correctly. An example of the result can be seen [here][ruby-style-guide.pdf].

  [ruby-style-guide.pdf]: https://github.com/rubocop-hq/ruby-style-guide/files/2392578/ruby-style-guide.pdf "ruby-style-guide.pdf"
